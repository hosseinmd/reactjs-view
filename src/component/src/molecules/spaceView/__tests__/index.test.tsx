import { shotTest } from "reactjs-view-tests";
import { SpaceView } from "../index";

shotTest(SpaceView, {
  children: (
    <>
      {null}
      <div />
      <>
        <div>
          <p />
        </div>
      </>
    </>
  ),
});

shotTest(SpaceView, {
  children: (
    <>
      {null}
      <div />
      <></>
      <>
        <div>
          <p />
        </div>
      </>
    </>
  ),
  spaceAround: true,
});
