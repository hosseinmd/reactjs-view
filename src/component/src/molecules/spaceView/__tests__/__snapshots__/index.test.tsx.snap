// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount and unmount component could be updated and unmounted without errors 1`] = `
<DocumentFragment>
  <div
    class="container-0-2-1"
  />
</DocumentFragment>
`;

exports[`mount and unmount component could be updated and unmounted without errors 2`] = `
<DocumentFragment>
  <div
    class="container-0-2-1"
  >
    <div />
    <div
      class="container-0-2-1"
    />
    <div>
      <p />
    </div>
  </div>
</DocumentFragment>
`;

exports[`mount and unmount component could be updated and unmounted without errors 3`] = `
<DocumentFragment>
  <div
    class="container-0-2-1"
  />
</DocumentFragment>
`;

exports[`mount and unmount component could be updated and unmounted without errors 4`] = `
<DocumentFragment>
  <div
    class="container-0-2-1"
  >
    <div
      class="container-0-2-1"
    />
    <div />
    <div
      class="container-0-2-1"
    />
    <div>
      <p />
    </div>
    <div
      class="container-0-2-1"
    />
  </div>
</DocumentFragment>
`;
